common/vendor/trezor-connect.js

```diff
@@ -1,5 +1,6 @@
 /* prettier-ignore */
 /* eslint-ignore */
+/* eslint-disable */
```

Comment:
We should probably just exclude everything in `/vendor/` at the eslintrc level, that's my bad for not doing so.